subfun/BIDS_RFX.m

```diff
@@ -22,14 +22,17 @@ function BIDS_RFX(action, mmFunctionalSmoothing, mmConSmoothing, opt)
 % The Contrast names should match those in the single level FFX and in THE
 % SAME ORDER.
 
+% Get the current working directory
+WD = pwd;
+
 % if input has no opt, load the opt.mat file
 if nargin<4
-    load('opt.mat') 
```

Comment:
the opt.mat is in the upper folder. 
so I go one folder up, load the opt.mat and return to the current WD again.

Reply:
its not pwd. 
The scripts are in subfunc folder, and the opt.mat is in the upper directory (not the current one). So we we need to load the opt.mat from the upper directory.

Or we make the getOptions function save the opt.mat in the subfunc folder.

Reply:
i Agree with you. Jumping across directories is not ideal.